etna/stories/models.py

```diff
@@ -44,52 +44,52 @@ class InsightsIndexPage(TeaserImageMixin, MetadataPageMixin, BasePage):
 
     def get_context(self, request):
         context = super().get_context(request)
-        insights_pages = self.get_children().public().live().specific()
-        context["insights_pages"] = insights_pages
+        stories_pages = self.get_children().public().live().specific()
+        context["stories_pages"] = stories_pages
         return context
 
     content_panels = BasePage.content_panels + [
         FieldPanel("sub_heading"),
-        FieldPanel("featured_insight"),
+        FieldPanel("featured_story"),
         FieldPanel("featured_pages"),
     ]
 
     promote_panels = MetadataPageMixin.promote_panels + TeaserImageMixin.promote_panels
 
-    subpage_types = ["insights.InsightsPage"]
+    subpage_types = ["stories.StoriesPage"]
 
 
 @register_snippet
-class InsightsTag(TagBase):
+class StoriesTag(TagBase):
     free_tagging = False
 
     class Meta:
-        verbose_name = "insights tag"
-        verbose_name_plural = "insights tags"
+        verbose_name = "stories tag"
+        verbose_name_plural = "stories tags"
 
 
-class TaggedInsights(ItemBase):
+class TaggedStories(ItemBase):
```

Comment:
```suggestion
class TaggedStory(ItemBase):
```

Reply:
This is now changed to reflect categorised name `Article`